### PR TITLE
Move some CI-related scripts to github

### PR DESCRIPTION
**Pull request by tavplubix**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Detailed description / Documentation draft:

1. Now tests tasks can write results into `test_results.tsv` file and check status into `check_status.tsv`. The first file should contain two or three columns (test name, result (`OK` or `FAIL`) and test time), each row is a result of some test. The second file should contain one row with check status to show on github, the first column is a status (`success` for green tick or `failure` for red cross) and the second column is text description. Task can put any logs into `/test_output/` dir. All files from `/test_output/` will be uploaded to s3 and attached to the report.
2. Part of integration tests runner moved from internal Yandex repository to GitHub
3. Check for hung queries is enabled for stress tests.

https://nda.ya.ru/t/hXTUekXP3krdjb

### CODE REVIEW

**Comment by tavplubix** [comment]
Stress test (address) - #21004
Stress test (thread) - something strange happened...

**Comment by tavplubix** [comment]
Stress test (memory) - `ALTER TABLE mt_00763_1 MODIFY column s String;` hung waiting for mutation

**Comment by tavplubix** [comment]
d5ea345aa9df9c77c95f9e5952251884407ae663:
Stress test (address) - `SYSTEM SYNC REPLICA` hung waiting on  `StorageReplicatedMergeTree::waitForShrinkingQueueSize`
Stress test (thread) - `SYSTEM RELOAD DICTIONARIES` hung inside `LoadingDispatcher::loadImpl`

**Comment by tavplubix** [comment]
Stress test (address) - `ALTER TABLE mt_00763_1 MODIFY column s UInt32` hung waiting for mutation - `StorageMergeTree::waitForMutation`
Stress test (thread) - `WATCH test.lv EVENTS` -  #21004
Stress test (undefined) - `ALTER TABLE collapsing_00682 DELETE WHERE k IN (1, 2)` and `alter table lc_empty_part_bug modify column s LowCardinality(String);`

**Comment by tavplubix** [comment]
f41e68d:
Stress test (address) - `alter table lc_empty_part_bug modify column s LowCardinality(String)` - `StorageMergeTree::waitForMutation`
Stress test (thread) - `SELECT 2, count() FROM system.numbers` - bad test `00600_replace_running_query`
Stress test (undefined) - `2021.02.24 01:17:42.862194 [ 3599 ] {3816e576-c98e-47e8-8ab7-105aeb6978a8} <Error> TCPHandler: Code: 49, e.displayText() = DB::Exception: The old table was not completely removed from ZooKeeper, /clickhouse/tables/test_00993/alter_table still exists and may contain some garbage. But it should never happen according to the logic of operations (it's a bug).`

3632c1d:
Stress test (address) - `ALTER TABLE replicated_mutation_table MODIFY COLUMN value UInt64 SETTINGS replication_alter_partitions_sync = 2`
Stress test (thread) - `SELECT 2, count() FROM system.numbers`

**Comment by tavplubix** [comment]
Stress test (thread) - OOM

File: docker/test/unit/process_unit_tests_result.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+
+import os
+import logging
+import argparse
+import csv
+
+OK_SIGN = 'OK ]'
+FAILED_SIGN = 'FAILED  ]'
```
Comment (alexey-milovidov) [inline docker/test/unit/process_unit_tests_result.py:9]: Double whitespace.

Comment (tavplubix) [reply docker/test/unit/process_unit_tests_result.py:9]: As it should be. Output of gtest looks like this:
```
[----------] 4 tests from ColumnUnique
[ RUN      ] ColumnUnique.InsertRange
[       OK ] ColumnUnique.InsertRange (5810 ms)
[ RUN      ] ColumnUnique.InsertRangeWithOverflow
../src/Columns/tests/gtest_column_unique.cpp:83: Failure
Expected equality of these values:
  idx->size()
    Which is: 1000000
  num_values + 1
    Which is: 1000001
[  FAILED  ] ColumnUnique.InsertRangeWithOverflow (5937 ms)
```

**Comment by tavplubix** [comment]
Docs check - broken in master
Functional stateless tests flaky check - 00600_replace_running_query - looks unrelated, the test was bad before current changes
Integration tests (thread) - test_zookeeper_config/test.py::test_secure_connection - flaky

**Pull request merged**